### ISSUE TITLE
Dashing does not work with IE9

### ISSUE THREAD

**Issue by arabold** [fabiocaseri/dashing-js#14]
Not much to add to the description. There're a couple of issues with IE9 support - for example EventSource is not supported and requires a Polyfill, and the CSS seems to be broken and not working.

I'm working on a patch right now. This task is for tracking and notifying you about the efforts and progress.

**Comment by fabiocaseri** [comment]
Yes, I never tested with IE because AFAIK it doesn't support Server-Sent Events and I can use chrome on my wall screen.
Ok, thanks!

**Comment by adrianlee** [comment]
I used https://github.com/Yaffle/EventSource/ polyfill to get the SSE working. CSS seem to be fine but no data seems to be displayed on the widgets. I'm also looking into this issue. @arabold any progress?

**Comment by arabold** [comment]
That's basically as far as I came, too. But I'm trying to dig deeper into it today and can hopefully submit a pull-request soon.

**Comment by adrianlee** [comment]
I got to familiarize myself with the IE dev tools. Do you have any idea if it's the dashing js or or batman?

**Comment by arabold** [comment]
It doesn't load the html of the widgets. I'm digging through the code right now... somewhat hard for me to understand the logic as I'm new to most of the modules used.

**Comment by arabold** [comment]
Okay, found it. `this.constructor.name` doesn't exist in IE and returns an `undefined`. This makes everything else fail badly. I'll submit a pull-request as soon as I got everything cleaned up again.

**Comment by arabold** [comment]
Here you go: https://github.com/fabiocaseri/dashing-js/pull/15 -- probably you should test that yourself, @adrianlee.

**Comment by fjorgemota** [comment]
@fabiocaseri What about using [socket.io](http://socket.io/) for get this compatibility? (it seems great for something like a dashboard, but it may need to refactor some core parts of the project, too)
